### PR TITLE
Desktop: closes#2894

### PR DESCRIPTION
**Pull request by infinity052**
conflicts were fixed and changes were added to AceEditor.tsx

### CODE REVIEW

File: ElectronClient/gui/style/theme/nord.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ const nordStyle = {
 	htmlCodeBorderColor: nord[2],
 	htmlCodeColor: nord[13],
 
-	editorTheme: 'chaos',
+	editorTheme: 'terminal',
```
Comment (laurent22) [inline ElectronClient/gui/style/theme/nord.js:80]: In AceEditor.tsx, you've added the "tomorrow" theme, and "twilight" was already present, however you didn't add "terminal". Isn't it an issue?

Comment (infinity052) [reply ElectronClient/gui/style/theme/nord.js:80]: yeah thats my bad i apologize for that. ive added it in the new commit.

**Comment by laurent22** [comment]
All good now, thanks @infinity052!

**Comment by laurent22** [comment]
I've reverted this pull request as the app doesn't even start:

image

It's not mentioned anywhere but I think it's obvious that a minimal requirement for a pull request is that the app starts at all. And this was very easy to check.

6446a3c4def2510762f1366000659fadd50b8ad4

**Comment by infinity052** [comment]
sorry, slight path issue in terminal import. #3192 final push.

**Pull request merged**